sites.json

```diff
@@ -2350,6 +2370,16 @@
     },
 
     {
+        "name": "gPodder",
```

Comment:
I made and pushed the requested changes. Let me know if there is anything else needed - thank you!